android/filament-android/src/main/java/com/google/android/filament/Engine.java

```diff
@@ -162,6 +162,8 @@ public enum FeatureLevel {
      * The type of technique for stereoscopic rendering
      */
     public enum StereoscopicType {
+        /** No stereoscopic rendering. */
+        NONE,
         /** Stereoscopic rendering is performed using instanced rendering technique. */
         INSTANCED,
         /** Stereoscopic rendering is performed using the multiview feature from the graphics backend. */
```

Comment:
there should be a not here that this value needs to match the materials used.

Reply:
done

Reply:
done.